server/server.js

```diff
@@ -19,41 +23,45 @@ const distPath = path.resolve(__dirname, '..', 'dist');
 const store = makeStore();
 const RoutingContext = createFactory(RoutingContextComponent);
 const Provider = createFactory(ProviderComponent);
+const verify = (token, callback) => {
+  return jwtoken.verify(token, 'secret', {}, callback);
+};
 
 const MongoDB = mongoose.connect(process.env.MONGOLAB_URI || config.developmentDB).connection;
 MongoDB.on('error', (err) => {
   console.log('Failed to connect go Mongo DB');
   console.log(err.message);
 });
-
 MongoDB.once('open', () => {
   console.log('Connected to Mongo DB');
 });
 
-console.log('Server running...');
-
 app.use(express.static(distPath));
```

Comment:
Is this the reason requests to root path don't reach the next middleware? We just serve the static file?